packages/nimble-components/src/dialog/tests/dialog.stories.ts

```diff
@@ -87,6 +126,11 @@ const metadata: Meta<DialogArgs> = {
         }
     },
     args: {
+        title: 'Dialog title',
```

Comment:
It would be good to add descriptions for title and subtitle that mention their interaction with ARIA and headerHidden. Here's a stab at it:

**title** Primary text that is displayed in the header when `headerHidden` is not set. Dialogs should always include a `title` even when `headerHidden` is set. The dialog includes `title` as part of `aria-labelledby` to provide a dialog label in assistive technologies regardless of the value of `headerHidden`.